content/docs/command-reference/import.md

```diff
@@ -91,13 +91,32 @@ from the source repo.
   > [Importing and updating fixed revisions](#example-importing-and-updating-fixed-revisions)
   > example below).
 
+- `-b`, `--backup` - imports files in "backup mode": see Backup Mode section for
+  details.
+
 - `-h`, `--help` - prints the usage/help message, and exit.
 
 - `-q`, `--quiet` - do not write anything to standard output. Exit with 0 if no
   problems arise, otherwise 1.
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
+## Backup mode
+
+To avoid duplicating large files, `dvc import` does not push imported files to
+any [remotes](/doc/command-reference/remote) by default. This means that when
+`dvc pull/fetch` is run, DVC retrieves imports from their source.
+So if the original source of an import is deleted, then the
+files are not backed up in your remote, and data may be lost.
+
+To prevent this, you can use the `--backup` option to request DVC to backup the
+data imports to remote storage. This also means that `dvc pull/fetch` no
+longer needs to access import sources, so
+these imports can be restored from remote storage like typical
+data tracked with `dvc add`.
+
+To update backed-up imports, use `dvc update` like usual.
+
```

Comment:
Let's put this whole section as an H3 inside the **Description**, just before **Options**, please.

Reply:
- [ ] Also, it should probably briefly describe what happens in the .dvc file (is there a new field?) and how to remove backup mode.
- [ ] Info about this field should also be added to https://dvc.org/doc/user-guide/dvc-files-and-directories#dvc-files please.

Reply:
Done and done. I already had some text in content/docs/user-guide/dvc-files-and-directories.md, though not loads. Is that fine?

Reply:
Great, thanks. I didn't see those changes in the other file, yes that's enough! I'll take it from there 🙂